mne/viz/misc.py

```diff
@@ -520,6 +519,7 @@ def plot_bem(subject=None, subjects_dir=None, orientation='coronal',
     on top of the midpoint MRI slice with the BEM boundary drawn for that
     slice.
     """
+    from .._freesurfer import _check_mri
```

Comment:
It doesn't seem like this should need to be nested, does it actually need to be?

Reply:
The other one is, I'll pull them both out